azure-quantum/azure/quantum/__init__.py

```diff
@@ -8,5 +8,7 @@
 from .job import *
 from .workspace import *
 
+from ._client.models import *
```

Comment:
```suggestion
from ._client.models._quantum_client_enums import JobStatus
```

Reply:
I think it's ._client.models import JobStatus (there's no _quantum_client_enums module). 
Let me make the new suggestion

Reply:
Yes, it is also available via _client.models, but the class is defined here: https://github.com/microsoft/qdk-python/blob/main/azure-quantum/azure/quantum/_client/models/_quantum_client_enums.py#L36

Reply:
ok changed